WearTilesKotlin/app/src/main/java/com/example/wear/tiles/messaging/MessagingRepo.kt

```diff
@@ -40,15 +40,15 @@ class MessagingRepo(private val context: Context) {
     suspend fun updateContacts(contacts: List<Contact>) {
         context.dataStore.edit {
             it.clear()
-            knownContacts.forEachIndexed { index, contact ->
+            contacts.forEachIndexed { index, contact ->
```

Comment:
I think a bug. No change in behavior since `knownContacts` is what gets passed to this function from the outside, but for correctness, this function doesn't know about the `knownContacts` property.